### PR TITLE
Add AutoIt

### LINKED ISSUE #10148: AutoIt Script
### Brand Name

AutoIt Scripting Language

### Website

https://www.autoitscript.com

### Popularity Metric

Similarweb: 263,633

### Official Resources for Icon and Color

svg: https://www.autoitscript.com/images/logo_autoit_210x72.svg
color: #254768

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by hatelamers**
autoitscript

**Issue:** closes #10148

**Similarweb rank:**
263,633

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
s. request #10148

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1551,6 +1551,11 @@
             "hex": "334455",
             "source": "https://www.autohotkey.com"
         },
+        {
+            "title": "AutoIt Script",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1555]: It looks like the brand name is actually just "AutoIt".
```suggestion
            "title": "AutoIt",
```

File: icons/autoitscript.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>AutoIt Script</title><path d="m19.351 15.563-5.486-7.941a2.684 2.684 0 0 0-.702-.702c-.276-.188-.62-.283-1.03-.283-.43 0-.784.101-1.064.302-.28.202-.512.43-.696.683l-5.63 7.94h3.215l4.122-5.827 1.575 2.323c.148.21.304.436.466.676.161.24.304.44.426.597a9.106 9.106 0 0 0-.741-.026H10.78l-1.64 2.258zM12 24C5.373 24 0 18.627 0 12S5.373 0 12 0s12 5.373 12 12-5.373 12-12 12zm0-21.61a9.61 9.61 0 1 0 0 19.22 9.61 9.61 0 1 0 0-19.22z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/autoitscript.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>AutoIt</title><path d="m19.351 15.563-5.486-7.941a2.684 2.684 0 0 0-.702-.702c-.276-.188-.62-.283-1.03-.283-.43 0-.784.101-1.064.302-.28.202-.512.43-.696.683l-5.63 7.94h3.215l4.122-5.827 1.575 2.323c.148.21.304.436.466.676.161.24.304.44.426.597a9.106 9.106 0 0 0-.741-.026H10.78l-1.64 2.258zM12 24C5.373 24 0 18.627 0 12S5.373 0 12 0s12 5.373 12 12-5.373 12-12 12zm0-21.61a9.61 9.61 0 1 0 0 19.22 9.61 9.61 0 1 0 0-19.22z"/></svg>
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -1551,6 +1551,11 @@
...
+            "hex": "254768",
+            "source": "https://commons.wikimedia.org/wiki/File:AutoIt-logo.svg"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1557]: As the Wikimedia file is just the website logo, we should use the website as the `source`.
```suggestion
            "source": "https://www.autoitscript.com"
```

**Review by PeterShaggyNoble — APPROVED** [review]
Thanks for the contribution, @hatelamers 👍🏻 SVG looks good to me. Just a couple of small changes to the icon data below and this should be ready to merge.

Happy to run with your choice of colour but I do wonder if the blue for the source SVG might not be a better option? 🤔

File: _data/simple-icons.json (hunk 3 above)
Comment (hatelamers) [reply _data/simple-icons.json:1557]: hmm, I used the Wikimedia as source and it clarifies copyright questions right on the spot, don't you think?

**Comment by hatelamers** [comment]
> Happy to run with your choice of colour but I do wonder if the blue for the source SVG might not be a better option? 🤔
The blue is from homepage head, which one do you mean?

File: _data/simple-icons.json (hunk 3 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json:1557]: We only resort to Wikimedia as a `source` in the absence of any other official vector artwork, or raster artwork that can be converted to vector.

**Comment by PeterShaggyNoble** [comment]
> The blue is from homepage head, which one do you mean?

`#5d83ac` from the `circle` in the [SVG](https://www.autoitscript.com/images/logo_autoit_210x72.svg).

File: _data/simple-icons.json (hunk 3 above)
Comment (hatelamers) [reply _data/simple-icons.json]: done

**Pull request merged**